runbotics/runbotics-desktop/src/action/sap/sap.action-handler.ts

```diff
@@ -45,29 +52,57 @@ export default class SapActionHandler extends StatefulActionHandler {
         return {};
     }
 
+    /**
+     *  @name SAP: End Transaction
+     *  @description Closes current transaction associated with session.
+     */
     async endTransaction() {
         await this.session?.EndTransaction();
     }
 
+
+    /**
+     *  @name SAP: Type text
+     *  @description Inserts text into the input field.
+     *  @param target - ID of SAP user interface element
+     *  @param value - text to be inserted
+     *  @example target: wnd[0]/example/target/path
+     *  @example value: example text
+     */
     async type(input: SapTypes.SAPTypeActionInput): Promise<SapTypes.SAPTypeActionOutput> {
         this.isApplicationOpen();
         const result = (this.session.FindById(input.target).text = input.value);
         return {};
     }
 
+    /**
+     *  @name SAP: Click
+     *  @description Clicks on the indicated element.
+     *  @param target - ID of SAP user interface element
+     *  @example target: wnd[0]/tbar[0]/btn[0]
+     */
     async click(input: SapTypes.SAPClickActionInput): Promise<SapTypes.SAPClickActionOutput> {
         this.isApplicationOpen();
         this.session.FindById(input.target).press();
         return {};
     }
 
+    /**
+     *  @name SAP: Double click
+     *  @description Clicks twice on the indicated element.
+     *  @param target - ID of SAP user interface element
+     *  @example target: wnd[0]/tbar[0]/btn[0]
+     */
     async doubleClick(input: SapTypes.SAPClickActionInput): Promise<SapTypes.SAPClickActionOutput> {
         this.isApplicationOpen();
         await this.focus(input);
         await this.sendVKey({ virtualKey: 'F2' });
         return {};
     }
 
+    /**
+     * @description This action should be used to handle the SAP table. It currently needs to be reworked.
```

Comment:
just `Reads SAP table`
add info about deprecation there should be a ready to use keyword